src/main/java/seedu/duke/lists/Workout.java

```diff
@@ -0,0 +1,39 @@
+package seedu.duke.lists;
+
+import seedu.duke.exception.GetJackDException;
+import seedu.duke.tasks.Exercise;
+
+import java.util.ArrayList;
+
+/**
+ * Adding, deleting and other operations related to the exercises will be executed from here.
+ */
+public class Workout {
+    public static ArrayList<Exercise> exercises;
```

Comment:
(I might be wrong) but I think exercises should be an instance and not static, since for each workout there are different lists of exercises